.github/workflows/check.yml

```diff
@@ -32,44 +31,61 @@ jobs:
         - 3.8
         - 3.7
         - 3.6
+        - 3.5
+        - pypy3
         - 2.7
         - pypy2
-        - pypy3
-
     steps:
```

Comment:
How about
```suggestion
    env:
      TOXENV: python

    steps:
```

Reply:
That wouldn't work and especially wouldn't active the conditional setenv parts.

Reply:
@gaborbernat it should work unless I'm overlooking something. I use it all the time. It only requires `actions/setup-python` to use a var from the matrix: https://github.com/ansible/pylibssh/blob/c13f2f5/.github/workflows/build-test-n-publish.yml#L358. And no dynamic TOXENV mapping needed at all.

Reply:
Oh, are you referring to the conditionals in `tox.ini`? Missed that.

Reply:
Yup.